bootstrap/cmd/kfctl/cmd/generate.go

```diff
@@ -15,91 +15,26 @@
 package cmd
 
 import (
-	"errors"
+	"fmt"
 
-	kftypes "github.com/kubeflow/kubeflow/bootstrap/v3/pkg/apis/apps"
-	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 )
 
-var generateCfg = viper.New()
-
+var generateDeprecationMessage = DeprecatedPrint("'kfctl generate' has been DEPRECATED.") + "\n" +
+`Please switch to new semantics.
+To build a KFAPP run -> ` + DeprecatedPrint("kfctl build -f ${CONFIG}") + "\n" +
+`Then to install -> ` + DeprecatedPrint("kfctl apply") + "\n" +
```

Comment:
What does the function DeprecatedPrint do? 

kfctl apply -> kfctl apply -f ${CONFIG}

Reply:
It prints out in color to add more "context" to the message.

Screenshot:
<img width="1495" alt="Screen Shot 2019-10-19 at 17 48 20" src="https://user-images.githubusercontent.com/23024031/67153125-ae9d3580-f298-11e9-87d7-343e6b677607.png">